docs/include/slack.txt

```diff
@@ -1,6 +1,6 @@
 Welcome to the Write the Docs Slack! We're glad you're here, and we look forward to your contributions to our community.
 
-We've written some guidelines to help you get the most out of this Slack. Please take a few minutes to learn about some of our channels, and familiarize yourself with our Slack guidelines. All this information is at <https://www.writethedocs.org/slack/> -- make sure to look through the whole page. We are grounded in the principles of open source, and we pride ourselves on being one of the most welcoming communities in tech. Our guidelines are an important tool to further this goal!
+We've written some guidelines to help you get the most out of this Slack. Please take a few minutes to learn about some of our channels, and familiarize yourself with our Slack guidelines. All this information is on [our Slack page](https://www.writethedocs.org/slack/) -- make sure to look through the whole page. We are grounded in the principles of open source, and we pride ourselves on being one of the most welcoming communities in tech. Our guidelines are an important tool to further this goal!
```

Comment:
I'm not sure if this is supposed to be in rst or markdown :-D

Reply:
Oh gosh, that was pure instinct on my part—sorry!